### DISCUSSION TITLE
Intro to Soroban

### DISCUSSION THREAD

**Discussion by theboycoder** [stellar/sorobanathon#153] [General]
I created and wrote a introduction post explaining Soroban. It's helpful introduction for beginners and someone unfamiliar with smart contracts. 

[https://github.com/theboycoder/Intro-to-Soroban](https://github.com/theboycoder/Intro-to-Soroban)

**Comment by rawritude** [comment]
Loving these soroban graphics!!!

**Comment by achallagundla** [comment]
Woah this is awesome, would love see this in a medium article format to share easily as a quick fact sheet on Soroban! 
Congratulations! Your contribution to Sorobanathon: First Light has been accepted! 🎉
Your submission may qualify for a reward of 1500 XLM! One last step: to receive that reward, please fill out this [form](https://stellarform.typeform.com/Sorobanathon) completely and accurately.

Please let us know if you have any questions. (You can also message me on discord @ anuxhya)

All the best,
Anuhya

PS: No need to submit the form if you already have, just update me on discord if your wallet address has changed.
